### PR TITLE
fix: Moves the rest of the Incoming packets

### PR DESCRIPTION
**Pull request by paezao**
### Summary
Moves the remainder of the incoming packets.

### CODE REVIEW

File: Projects/Server/Gumps/Gump.cs [hunk 1]
```diff
@@ -15,7 +15,19 @@ public partial class Gump
...
+    {
+        get => m_Switches;
+        set => m_Switches = value;
+    }
+
+    public int TextEntries
+    {
+        get => m_TextEntries;
+        set => m_TextEntries = value;
+    }
```
Comment (kamronbatman) [inline Projects/Server/Gumps/Gump.cs:30]:
```suggestion
    public int Switches { get; set; }
    public int TextEntries { get; set; }
```

File: Projects/Server/MessageType.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
+﻿using System;
```
Comment (kamronbatman) [inline Projects/Server/MessageType.cs:1]: Add copyright. If you are using Rider, you can import rider-settings.zip and do `copyright` and select it from the autocomplete dropdown to generate it.

File: Projects/Server/Gumps/Gump.cs [hunk 3]
```diff
@@ -15,7 +15,19 @@ public partial class Gump
     public static readonly byte[] NoDispose = StringToBuffer("{ nodispose }");
     public static readonly byte[] NoResize = StringToBuffer("{ noresize }");
 
-    internal int m_TextEntries, m_Switches;
+    private int m_Switches, m_TextEntries;
```
Comment (kamronbatman) [inline Projects/Server/Gumps/Gump.cs:18]: Let's rename them to use the new convention of _switches, etc.

**Review by kamronbatman — COMMENTED** [review]
Looks good just a few minor things.

File: Projects/Server/MessageType.cs (hunk 2 above)
Comment (paezao) [reply Projects/Server/MessageType.cs]: Added!

File: Projects/Server/Gumps/Gump.cs (hunk 3 above)
Comment (paezao) [reply Projects/Server/Gumps/Gump.cs]: Done!

File: Projects/Server/Gumps/Gump.cs [hunk 4]
```diff
@@ -15,7 +15,19 @@ public partial class Gump
     public static readonly byte[] NoDispose = StringToBuffer("{ nodispose }");
     public static readonly byte[] NoResize = StringToBuffer("{ noresize }");
 
-    internal int m_TextEntries, m_Switches;
+    private int _switches, _textEntries;
```
Comment (kamronbatman) [inline Projects/Server/Gumps/Gump.cs:18]:
```suggestion
    private int _switches;
    private int _textEntries;
```

**Review by kamronbatman — APPROVED** [review]

**Pull request merged**